cmd/snapshot-controller/main.go

```diff
@@ -74,46 +74,49 @@ var (
 	preventVolumeModeConversion   = flag.Bool("prevent-volume-mode-conversion", true, "Prevents an unauthorised user from modifying the volume mode when creating a PVC from an existing VolumeSnapshot.")
 	enableVolumeGroupSnapshots    = flag.Bool("enable-volume-group-snapshots", false, "Enables the volume group snapshot feature, allowing the user to create a snapshot of a group of volumes.")
 
-	retryCRDIntervalMax = flag.Duration("retry-crd-interval-max", 5*time.Second, "Maximum retry interval to wait for CRDs to appear. The default is 5 seconds.")
+	retryCRDIntervalMax = flag.Duration("retry-crd-interval-max", 10*time.Second, "Maximum time to wait for CRDs to appear. The default is 10 seconds.")
```

Comment:
Would 30s be a better default? Sometimes CRDs and the controller are all started at the same time, so a bit more slack on allowing everything to come up would be better.

This longer delay might only make sense if we have a readiness probe --- will a 30s default cause problems for helm as you mentioned in #981?

Reply:
30s works for me. This shouldn't cause any problems related to #981 as long as `minReadySeconds` is above this value in the chart. Like this helm (or whatever deployment system waiting for pod readiness) will wait at least `minReadySeconds` before moving on.

I do think that adding a readiness probe is a better way to achieve the goal of 'waiting some time before sending the pod in crashloop'. I don't mind making another PR to add one so that we can maybe deprecate the flag `retryCRDIntervalMax` and just rely on the probe during startup.